src/js/rx/actions/prescriptions.js

```diff
@@ -1,4 +1,25 @@
-export function loadData() {
+export function loadData(id) {
+  if (id) {
+    // TODO: Use id param instead of test id
+    // when API is able to retrieve any individual Rx.
+    const testId = 1435525;
+    return dispatch => {
+      fetch(`/api/prescriptions/${testId}`)
+      .then(res => res.json())
+      .then(
+        data => dispatch({ type: 'LOAD_PRESCRIPTION_SUCCESS', data }),
+        err => dispatch({ type: 'LOAD_PRESCRIPTION_FAILURE', err })
+      );
+
+      fetch(`/api/prescriptions/${testId}/trackings`)
+      .then(res => res.json())
+      .then(
+        data => dispatch({ type: 'LOAD_RX_TRACKINGS_SUCCESS', data }),
+        err => dispatch({ type: 'LOAD_RX_TRACKINGS_FAILURE', err })
+      );
```

Comment:
Is it okay that each of these calls come back at different times?  We should be careful for avoid a situation where it's showing tracking information for the wrong prescription...